readme.md

```diff
@@ -2,7 +2,16 @@
 
 <sup>(browser + oyster)</sup>
 
-[WIP] A monorepo of testing tools
+<p align="center">
+  <a href="https://github.com/fingerprintjs/broyster/actions/workflows/test.yml">
+    <img src="https://github.com/fingerprintjs/broyster/actions/workflows/test.yml/badge.svg" alt="Build status">
+  </a>
+  <a href="https://www.npmjs.com/package/@fpjs-incubator/broyster">
+    <img src="https://img.shields.io/npm/v/@fpjs-incubator/broyster.svg" alt="Current NPM version">
+  </a>
+</p>
+
+A monorepo of testing tools for Karma with advanced usage of Browserstack and a Jasmine retry mechanism.
```

Comment:
I still don't understand what it actually does. Why did you make this? What problem were you trying to solve? What abilities does it give you that you didn't have before? Something like: 

"Broyster combines several testing tools together into a single workflow that allows you to easily test applications across repositories/in the cloud/using X/whatever: 
* uses Karma for A 
* uses Browsertack for B
* Jasmine as a retry mechanism
... 

Something like that, but actually true. Does that make sense? :)